blueprints/ember-redux/files/__root__/reducers/index.js

```diff
@@ -0,0 +1,18 @@
+import { combineReducers } from 'redux';
+
+/**
+ * This is the master reducer, it partitions the jobs amounts
+ * one or more other reducers which take on responsibility for
+ * a discrete part of the state tree.
+ *
+ * Use the "ember generate reducer" command to create additional
+ * reducers.
+ */
+
+// import foo from './foo';
+// import bar from './bar';
+
+export default combineReducers({
```

Comment:
I would think you would want this as the primary reducer in anything but a small app will just be handing off (aka, combining) to other reducers.